migtests/tests/pg/assessment-report-test-with-tdb/env.sh

```diff
@@ -1,4 +1,4 @@
 export SOURCE_DB_TYPE="postgresql"
 export SOURCE_DB_NAME=${SOURCE_DB_NAME:-"pg_assessment_report"}
 export SOURCE_DB_SCHEMA="public,schema2,test_views"
-export TARGET_DB_VERSION="2.25.0.0"
\ No newline at end of file
+export TARGET_DB_VERSION="2.25.1.0"
```

Comment:
Concerned about test coverage wrt to target db version aware issue reporting.
This test is checking verifying with one version, and keeps changing.

We need to test for multiple versions, ideally should add new version and keep previous ones unless voyager is deprecating the support for that YB version


Two solutions I see:
1. Modify this test/framework to run for multiple YB versions and keep old reports and add new report(copy/paste with changes in issuelist) for newer YB versions.

2. Think of shifting this to our go test framework. Not a major difference from test perspective but will be much easier to do it there and write custom logic for each test.

Reply:
I think this is okay for now, we don't have much difference between 2.25.1 and 2.25.0. Maybe we do it in a separate PR where we can unit test for different target databases. stable or preview cases and have some basic validations of a few things only.

Reply:
Please create a ticket for this.
We should test not just the one latest release but overall - latest stable, latest preview etc.. any important ones basically.

Reply:
Created a ticket https://yugabyte.atlassian.net/browse/DB-16692